src/main/java/no/nav/familie/integrasjoner/personopplysning/internal/PdlPersonRequest.kt

```diff
@@ -0,0 +1,4 @@
+package no.nav.familie.integrasjoner.personopplysning.internal
+
+data class PdlPersonRequest (val variables: PdlRequestVariable,
```

Comment:
Kunne generalisert  dette for våre graphql-klieneter. Med å ha felles klasser for GrapqlRequest, GraphqlResponse, GraphqlError.

Reply:
God ide! Kan vel ta den i en egen PR kanskje? Kan få inn en oppgave i favro på det.

Reply:
Ja. Samkjør litt med @geirmundsimonsen som skal røre safklient på mandag.